### ISSUE TITLE
Unable to load website

### ISSUE THREAD

**Issue by SkyNET-BIOS** [gryffon/ringteki#883]
When I try to access https://jigoku.online the site never loads up. On Firefox it hangs with "performing a TLS handshake to jigoku.online" and with a fresh, vanilla install of Chrome I get "ERR_TIMED_OUT". I have been able to access the beta site with no issues. 

![githubjigokuscnsht - chrome](https://user-images.githubusercontent.com/33473167/32529919-0ee6734a-c3f0-11e7-87d7-d0897e618508.PNG)

**Comment by SkyNET-BIOS** [comment]
I can access the live site with my phone's browser.

I haven't thought of anything to try with Chrome but with Firefox I checked my security.tls.version.max value and it was set to the default at 3. I tried setting that to 4 and 5 and tried accessing the site with no results.

**Comment by SkyNET-BIOS** [comment]
I used a online website to convert the domain name to it's IP address. I tested this with twitch.tv first to verify functionality and then got the IP address for jigoku.online which translated to 208.97.140.191. Putting this in Firefox or Chrome resulted in this being displayed:

![githubjigokuscnsht - ip address](https://user-images.githubusercontent.com/33473167/32584329-92c84abe-c4ac-11e7-9463-d1ce8ded2082.PNG)

I also get this page on my phone's browser (Firefox) despite that the site resolves normally with the regular hostname on the phone browser.
